pooltasks.md

```diff
@@ -98,12 +98,7 @@ Return codes:
 The creation of pool tasks is managed by the underline workflow system. No methods are exposed to manually trigger such creation to avoid workflow hjack and inconsistency.
 
 ## POST Method (single resource level)
-To claim a pool task a POST request must be issued against the single pool task URL
-/api/workflow/pooltasks/:id
-
-204 No content is returned if the request succeed
-403 Not authorized if the task cannot be claimed by the logged in user
-404 if the task is not longer available
+This method has been moved to the [claimed tasks](claimedtasks.md#post-method) resource.
```

Comment:
I'd recommend we reword this to say:
`Not allowed. To claim a pool task, please POST against the [claimed tasks](claimedtasks.md#post-method) endpoint.`

That's a clearer message than saying that a method "moved".  Once this tiny change is made, this all looks good

Reply:
@tdonohue : Thanks for the suggestion, I applied it to the branch 👍